library/ajax/drug_autocomplete/index.php

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Demo</title>
+  <link rel="stylesheet" href="../../library/js/jquery-ui.min.css" type="text/css" /> 
```

Comment:
remove lines 6, 14, 15 and then place following code here:
``````
<?php
html_header_show();
$include_standard_style_js = array("jquery-ui","jquery-ui-sunny");
require "{$GLOBALS['srcdir']}/templates/standard_header_template.php";
?>